lib/index.coffee

```diff
@@ -20,9 +22,27 @@ module.exports =
       loadModule()
       packageSync.sync()
 
+    atom.packages.onDidActivateInitialPackages ->
+      @didActivateInitialPackages = true
```

Comment:
Since the handlers are added whether the setting is enabled or not, if they enable or disable `createOnChange`, it should take effect "immediately".